log/rotation/file.go

```diff
@@ -83,11 +83,14 @@ func (f *File) write(p []byte) (int, error) {
 	}
 	if f.f == nil {
 		var err error
-		f.f, err = os.OpenFile(f.base, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0666)
+		f.f, err = os.OpenFile(f.base, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644) // #nosec
```

Comment:
I don't think it makes sense for the logfile to be world-writable, so I changed the perms to `0644`. At the very least I don't think they should be looser than `0664`.

I had to add `#nosec` anyway as it prefers `0600`, which we might also consider.